diesel/src/deserialize.rs

```diff
@@ -545,7 +545,9 @@ where
         use crate::row::Field;
 
         let field = row.get(0).ok_or(crate::result::UnexpectedEndOfRow)?;
-        T::from_nullable_sql(field.value())
+        T::from_nullable_sql(field.value()).map_err(|e| {
+            format!("Error for field '{}': {:?}", field.field_name().unwrap(), e).into()
```

Comment:
Given that this might contain a completely user defined error (as `FromSql` can be implemented by third party crates for custom types) I wouldn't want to just convert that error to a string. Rather I would prefer having an error type that wraps the inner error. So something like 

```rust
// part of `src/result.rs

#[non_exhaustive]
pub struct DeserializeFieldError {
    // use an option to not require unwrapping
    pub field_name: Option<String>,
    pub error: Box<dyn Error + …>
}

impl Error for DeserializeFieldError {
 // … (forward the inner error as source)
}
```

Afterwards we can change this line to use the new error type instead.